packages/kbn-plugin-generator/sao_template/template/server/plugin.ts

```diff
@@ -0,0 +1,21 @@
+
+import { PluginInitializerContext, CoreSetup, CoreStart, Plugin } from '<%= relRoot %>/src/core/server';
+
+export interface <%= camelCaseName %>PluginSetup {};
+export interface <%= camelCaseName %>PluginStart {};
```

Comment:
NIT: inconsistency: the setup/start interfaces got their own `types` file for `public` but are in the plugin file for `server`. both are fine, but public/server should have the same structure